handlers/smshandler/mongodbstorage.go

```diff
@@ -139,19 +139,20 @@ func (ms *MongoStorage) Users() (*Users, error) {
 }
 
 func (ms *MongoStorage) AddUser(userID types.HexBytes, processIDs []types.HexBytes,
-	phone, extra string) error {
+	phone, extra string,
+) error {
 	phoneNum, err := phonenumbers.Parse(phone, DefaultPhoneCountry)
 	if err != nil {
 		return err
 	}
 	ms.keysLock.Lock()
 	defer ms.keysLock.Unlock()
 
-	maxAttempts := ms.maxSmsAttempts * len(processIDs)
-	// nolint[:ineffassign]
-	if maxAttempts == 0 {
-		maxAttempts = ms.maxSmsAttempts
-	}
+	// maxAttempts := ms.maxSmsAttempts * len(processIDs)
+	// // nolint[:ineffassign]
+	// if maxAttempts == 0 {
+	// 	maxAttempts = ms.maxSmsAttempts
+	// }
```

Comment:
delete code rather than commenting it out, it's always in the git history

Reply:
Out of the PR, but yeah, agree.